docs/UserGuide.md

```diff
@@ -86,15 +86,87 @@ This marks the end of the quick start. Click
+
 ### 3.1 Special Icons
 
+Here are the special icons used throughout the User Guide to help you utilise the User Guide more effectively.
+
+#### Tips
+
+Tips are tidbits of helpful information that can help you have a better experience with DonnaFin.
+
+<div markdown="span" class="alert alert-primary">:bulb:
+Tip: Tips are helpful!
+</div><br>
+
+#### Notes
+
+Notes are here to provide the User with extra information that they may find useful when using DonnaFin.
+
+<div markdown="span" class="alert alert-primary">:information_source:
+Take notes when you see this icon!
+
+
+#### Caution 
+
+Cautions are around to warn users of potential unwanted side effects that may occur on certain parts of DonnaFin if the 
+user fails to follow the user guide or tamper with the files of the application.
+
+<div markdown="span" class="alert alert-primary">
+:exclamation: Be wary of cautions!
+</div><br>
+
 ### 3.2 Application Home Window Breakdown
 
-### 3.3 Client Explanation
+### 3.3 Application Client Window Breakdown
+
 
-### 3.4 Application Client Window Breakdown
+#### 3.3.4 Asset/Liabilities Tab
 
-### 3.5 Command Format
+###### This section covers both the asset and liability tab due to their similarities.
+
+The Asset/Liabilities tab is the Client View tab that stores all of your client's assets/liabilities. This tab can be
+accessed by pressing on the Assets/Liabilities tab when in any of the Client View tabs, or through typing the command
+`tab a`, `tab asset`, or `tab assets` for the Asset tab and typing the command `tab l`, `tab liability`, or
+`tab liabilites`.
+
+You will know that you are in the Asset tab if you see the **Assets** header below the Result Display or the Liability
+tab if you see the **Liabilities** header below the Result Display. You can see the total value of these assets/
+liabilities by looking at the number beside the **Total Asset/Liability Value**. 
+
+The Assets/Liabilities are displayed in a Table View where each row represents a new asset/liability, while each column
+represents a Field of an Asset/Liability.
+
+![Asset/Liabilities Tab of DonnaFin](images/Asset_Liability.png)
+
+Here are the following fields that each asset/liability records:
+1. Index
+2. Asset/Liability Name
+3. Asset/Liability Type
+4. Asset/Liability Value
+5. Remarks
+
+Each asset/liability has an index which is sorted by ascending order, and if you would like to remove an existing
+asset/liability , you can keep track of the index by looking at the number to the left of the asset/liability name you
+would like to delete.
+
+The Asset/Liability tab also supports the `append` and `remove` command that allows you to add a new asset/liability or
+remove an existing asset/liability as shown 
+[here.](https://ay2122s1-cs2103t-w16-1.github.io/tp/UserGuide.html#4331-add-an-asset-to-a-client-append.)
+
+#### 3.3.5 Notes Tab
+
+The Notes tab is the Client View tab that stores additional information about your client. This tab can be accessed by
```

Comment:
Client Window tab? consider : The Notes tab stores additional information about your client.